bin/projects/dbatools/msbuild.rsp

```diff
@@ -6,4 +6,5 @@
 /ConsoleLoggerParameters:NoSummary;Verbosity=quiet
 # Log to a file and set options
 /filelogger
-/fileloggerparameters:Verbosity=detailed
\ No newline at end of file
+/fileloggerparameters:Verbosity=detailed
+/logger:BinaryLogger,".\bin\StructuredLogger.dll";".\msbuild.binlog"
```

Comment:
Fun facts here:

* The file name needs to end in `.binlog`. 
* Relative paths need to be prepended to `.\` to be parsed.

Reply:
hey zips, how we lookin as far as progress and merging?